src/qibosoq/qick_programs.py

```diff
@@ -210,6 +219,19 @@ def body(self):
         self.wait_all()
         self.sync_all(self.relax_delay)
 
+    def is_pulse_equal(self, pulse_a: Pulse, pulse_b: Pulse) -> bool:
+        """Check if two pulses are equal, does not check the start time"""
+        if pulse_a is None or pulse_b is None:
+            return False
+        return (
+            pulse_a.frequency == pulse_b.frequency
+            and pulse_a.amplitude == pulse_b.amplitude
+            and pulse_a.relative_phase == pulse_b.relative_phase
+            and pulse_a.duration == pulse_b.duration
+            and pulse_a.type == pulse_b.type
+        )
+        # and pulse_a.shape == pulse_b.shape
```

Comment:
hi @stavros11, in this function I wanted to compare 2 pulses (but not the start time). 
Everything worked fine, expect the comparison between shapes that, for identical rectangular pulses was giving
`pulse_a.shape == pulse_b.shape` equal to `False`.
Do you know what could have cause it?